CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [1.4.0] - 2021-12-03
 ### Added
 - ACF plot ([#318](https://github.com/tinkoff-ai/etna/pull/318))
+- Holiday Transform ([#359](https://github.com/tinkoff-ai/etna/pull/359))
```

Comment:
it is CHANGELOG for release 1.4.0
you need to add [Unreleased] block if it doesn't exist

Reply:
Ooops